doc/sphinx/proof-engine/tactics.rst

```diff
@@ -202,28 +196,25 @@ For a goal :g:`A \/ B`, use :tacn:`left` to replace the current goal with :g:`A`
   of :n:`let-ins`, the patterns are applied to the arguments and :n:`let-in` variables.
 
 * :n:`( {+& @simple_intropattern} )` - matches a term that consists of one or more
-  right-hand nested binary inductive types in a form such as :g:`(a1 OP (a2 OP ...))`.
+  nested binary inductive types in a right-associative form such as :g:`(a1 OP (a2 OP ...))`.
```

Comment:
Maybe keep your old formulation but instead remove the parentheses from your example and add "(supposing that OP is right-associative)"...